force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -192,13 +225,17 @@ public with sharing class ClassesObjectsMethods {
     public static String getDayOfWeek(DateTime dt) {
         // Define a map of day of the week as a key and day number as the value
         // Monday = 1, Tuesday = 2, etc.
+        Map<String,Integer> numberByWeekdays = new Map<String,Integer>{'Monday' => 1, 'Tuesday' => 2, 'Wednesday' => 3, 'Thursday' =>4, 'Friday' => 5, 'Saturday' => 6, 'Sunday' =>7};
 
         // Get the day name using .format('EEEE');
+        String dtInString = dt.format('EEEE');
```

Comment:
This is another insanely important method to understand how it works and be aware that it does.